README.md

```diff
@@ -145,6 +126,36 @@ these two situations.
 > uv add https://github.com/FAIRmat-NFDI/nomad-measurements --branch <specific-branch-name>
 > ```
 
+To ensure `uv` recognizes the
+local plugins (a local copy of your plugin repository available in `packages/` directory),
+we need to make some modifications in the `pyproject.toml`.
+These include adding the plugin package to `[project.dependencies]` and
+`[tool.uv.sources]` tables.
+For the packages listed under `[tool.uv.sources]`, `uv` uses the local code
+directory made available under `packages/` with the previous
+step.
```

Comment:
Maybe still keep this paragraph at the top of step 3 to let the reader know why we do all this.

Reply:
Let me know if it's ok now

Reply:
Looks good